HW04/Ivan_Papushoi/test_methods.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'ivan_papushoi_hw04.rb'
+require 'rspec/autorun'
+
+RSpec.describe Array do
+  it 'check my_select return' do
+    array = [1, 2, 3, 4, 5, 6]
+    term = ->(x) { x.odd? }
+    expectation = [1, 3, 5]
```

Comment:
Normally, we don't use variables in spec everything should be in `let`
```suggestion
    let(:array) { [1, 2, 3, 4, 5, 6] }
    let(:term) { ->(x) { x.odd? } }
    let(:expectation) { [1, 3, 5] }
    it 'check my_select return' do

```

Reply:
Done it